tests/aib/check_qm_with_aib.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Create AIB manifest with the correct QM copr
+USE_QM_COPR="${PACKIT_COPR_PROJECT:-rhcontainerbot/qm}"
+sed -e "s,\${QM_COPR},${USE_QM_COPR},g" check_qm_with_aib.aib.yml > qm.aib.yml
```

Comment:
maybe one more thing...why doing it in such an ugly way? you can pass a variable by --define, or add extra_repo...see the example in https://sigs.centos.org/automotive/building/building_an_os_image_that%20uses_a_custom_kernel/#build-an-autosd-image-with-a-custom-kernel-and-run-it-on-physical-hardware

Reply:
Good point. Done